bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/ChannelTrigger.json

```diff
@@ -18,16 +18,25 @@
                      "name": "kind"
                   }
                ]
+            },
+            {
+               "name":"eventOnChannel",
+               "type":"TEXT",
+               "label":"Event",
+               "description":"the event on the channel to react on",
+               "required":false,
+               "defaultValue":""
```

Comment:
Why do you define a default? It is an optional parameter anyhow, so if not provided, it should be rather null.